Makefile

```diff
@@ -92,7 +92,7 @@ test-ruff:
 
 .PHONY: test-integrations
 test-integrations: ## Run integration tests.
-	pytest tests/integration
+	pytest --basetemp .tmp tests/integration
```

Comment:
Changed `basetemp` dir to avoid using `sudo` to write files to tempdir. See https://docs.pytest.org/en/stable/how-to/tmp_path.html#temporary-directory-location-and-retention